build_tools/mako/compile_android_modules.sh

```diff
@@ -50,8 +50,6 @@ if [[ -z "${model}" ]]; then
   exit 1
 fi
 
-export IREE_LLVMAOT_LINKER_PATH="${ANDROID_NDK?}/toolchains/llvm/prebuilt/linux-x86_64/bin/aarch64-linux-android29-clang++ -static-libstdc++ -O3"
```

Comment:
triggered a build to test this: https://buildkite.com/iree/iree-android-benchmark/builds/1821